docs/classes/PDFPage.rst

```diff
@@ -204,7 +200,27 @@ PDFPage
         let annots = getAnnotations();
         page.delete(annots[0]);
 
-    
+.. _Classes_PDFPage_deleteLink:
```

Comment:
If this is incorrectly named, just rename it. Users are able to update their code up on API changes. And this is not an API we want to abide by forever anyway.

If we really want to care about backwards compatibility, make a wrapper like you did for authenticate() and keep it around for a single release.